source/helpers/bisect.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'dom-chef';
+import cache from 'webext-storage-cache';
+import select from 'select-dom';
+
+import features from '../features';
+
+export default async function bisectFeatures(): Promise<Record<string, boolean>> {
+	const bisectedFeatures: string[] | undefined = await cache.get('bisect');
+	if (bisectedFeatures === undefined) {
+		return {};
+	}
+
+	const enabledFeatures = new Set(bisectedFeatures.slice(0, Math.ceil(bisectedFeatures.length / 2)));
+	if (bisectedFeatures.length === 0) {
+		createMessageBox(<div><p>Every feature has been disabled. Do you still see the change or issue?</p><p>Note: You can change page here, but don’t use multiple tabs.</p></div>);
+	} else if (bisectedFeatures.length === 1) {
+		createMessageBox(<p>The change or issue is caused by <a href={'https://github.com/sindresorhus/refined-github/blob/main/source/features/' + bisectedFeatures[0] + '.tsx'}><code>{bisectedFeatures[0]}</code></a>.</p>, false);
```

Comment:
Ideally this step should also be moved to onChoiceButtonClick, but I can see that it'd mean copying the `slice` there as well.

At a minimum, we can add
```suggestion
	} else if (bisectedFeatures.length === 1) {
		createMessageBox(<p>The change or issue is caused by <a href={'https://github.com/sindresorhus/refined-github/blob/main/source/features/' + bisectedFeatures[0] + '.tsx'}><code>{bisectedFeatures[0]}</code></a>.</p>, false);
		await cache.delete('bisect');
		return;
```